### ISSUE TITLE
[Bug] 图例的label的Space配置不生效

### ISSUE THREAD

**Issue by neuqzxy** [VisActor/VChart#3297]
### Version

1.12.6

### Link to Minimal Reproduction

1.12.6

### Steps to Reproduce

```
const data = [
  { value: 10, category: 'One' },
  { value: 9, category: 'Two' },
  { value: 6, category: 'Three' },
  { value: 5, category: 'Four' },
  { value: 4, category: 'Five' },
  { value: 3, category: 'Six' },
  { value: 1, category: 'Seven' }
];
let totalValue = 0;
data.forEach(obj => (totalValue += obj.value));
const map = {};
data.forEach(obj => {
  map[obj.category] = `${((obj.value / totalValue) * 100).toFixed(2)}%`;
});

const spec = {
  type: 'pie',
  data: [
    {
      id: 'pie',
      values: data
    }
  ],
  categoryField: 'category',
  valueField: 'value',
  legends: {
    visible: true,
    orient: 'bottom',
    data: items => {
      return items.map(item => {
        item.value = map[item.label];
        return item;
      });
    },
    item: {
      shape: {
        space: 10
      },
      label: {
        // space: 2000
      },
      value: {
        space: 50,
      }
    }
  }
};

const vchart = new VChart(spec, { dom: CONTAINER_ID });

vchart.on('legendItemHover', e => {
  const hoveredName = e?.value?.data?.label;
  if (hoveredName) {
    vchart.updateState({
      legend_hover_reverse: {
        filter: d => d.type !== hoveredName
      }
    });
  }
});
vchart.on('legendItemUnHover', e => {
  vchart.updateState({
    legend_hover_reverse: {
      filter: d => false
    }
  });
});

vchart.renderSync();

// Just for the convenience of console debugging, DO NOT COPY!
window['vchart'] = vchart;
```

### Current Behavior

space不正常

### Expected Behavior

space正常生效

### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by neuqzxy** [comment]
用户希望能在10月下旬前解决这个问题

**Comment by xile611** [comment]
fixed in 1.12.8
